### PR TITLE
added one seed/crop type

### PR DESCRIPTION
**Pull request by RUsami**
Added Elevibloom seed to the Crop quest

### CODE REVIEW

**Comment by RUsami** [comment]
Actually Updated a bunch of quest (agriculture), can we merge the two commit together ?

**Review by Jonah-Hansen — APPROVED** [review]

File: overrides/config/ftbquests/quests/chapters/all_things_food.snbt [hunk 1]
```diff
@@ -1350,6 +1383,25 @@
...
+							"dungeons_libraries:built_in_enchantments": { }
+						}
+<<<<<<< Updated upstream
+						id: "enlightened_end:elevibloom_seeds"
+=======
+						id: "atmospheric:aloe_kernels"
+>>>>>>> Stashed changes
+					}
+					type: "item"
+				}
```
Comment (Jonah-Hansen) [inline overrides/config/ftbquests/quests/chapters/all_things_food.snbt:1404]: these "<<<<<<< Updated upstream" lines are remenants of git merges, try removing it but double check that elevibloom seeds and aloe kernals still show up properly

**Review by Jonah-Hansen — CHANGES_REQUESTED** [review]

**Comment by Jonah-Hansen** [comment]
> Actually Updated a bunch of quest (agriculture), can we merge the two commit together ?

having multiple commits is totally fine. generally what we want is 
   1 commit = only the changes necessary for one small change/feature
   1 pull request = a collection of related commits

**Comment by Jonah-Hansen** [comment]
@RUsami so this chapter is being deleted with 1.5.4 rework. you'll have to add these to the crop quest in the `Quests` chapter instead after #224 is merged, or branch off that one to make the change.

**Comment by RUsami** [comment]
Actually, I think all the changes here are already in the Christi Big ftbquest config changes ? 
Sorry for not saying it earlier

**Comment by Jonah-Hansen** [comment]
> Actually, I think all the changes here are already in the Christi Big ftbquest config changes ? Sorry for not saying it earlier

I think you're right! It's all good then

**Pull request closed without merging**